test/scripts/with-dotenv/with-dotenv.yml

```diff
@@ -0,0 +1,6 @@
+config:
+  target: "https://www.wontresolve.none"
+scenarios:
+  - flow:
+      - get:
+          url: "{{ $processEnvironment.URL }}"
```

Comment:
Hello! When reading through the [Artillery.io documentation](https://www.artillery.io/docs/reference/cli/run#set-scenario-variables-on-the-fly) on the `--dotenv` flag, I couldn't find any information on how to actually reference the `.env` variables within the `artillery.yaml` file. I tried `{{ process.env.VAR }}`, `{{ $VAR }}, etc. After spending quite a bit of time guessing and checking, as well as asking AI (lol), I finally stumbled across this PR where I found you referencing the `.env` variables as `{{ $processEnvironment.VAR }}`. If possible, this could be super useful to include in the Artillery.io docs, to save other people the potential frustration of guessing and checking (maybe it's only me 😆 )

Reply:
thank you for the feedback @danfooks, appreciate you taking the time! pushing an update to the docs now

Reply:
Awesome @hassy !!! Thanks so much!